indexd/index/blueprint.py

```diff
@@ -374,6 +375,74 @@ def add_index_record_version(record):
 
     return flask.jsonify(ret), 200
 
+@blueprint.route("/index/<path:record>/aliases", methods=["GET"])
+def get_aliases(record):
+    """
+    Get all aliases associated with this DID / GUID
+    """
+    try:
+        aliases = blueprint.index_driver.get_aliases_for_did(record)
+        aliases_payload = {
+            "aliases": [{"value": alias} for alias in aliases]
+        }
+        return flask.jsonify(aliases_payload), 200
+    except NoRecordFound as err:
+        raise err
+    except UserError as err:
+        raise err
+
+@blueprint.route("/index/<path:record>/aliases", methods=["POST"])
+def append_aliases(record):
+    """
+    Append one or more aliases to aliases already associated with this
+    DID / GUID, if any.
+    """
+    aliases_json = flask.request.get_json()
+    try:
+        jsonschema.validate(aliases_json, RECORD_ALIAS_SCHEMA)
+    except jsonschema.ValidationError as err:
+        raise UserError(err)
```

Comment:
what does this error look like? does it clearly state that the body doesn't have the expected format?

Reply:
The jsonschema error looks like this: https://python-jsonschema.readthedocs.io/en/stable/errors/#jsonschema.exceptions.ValidationError.message
The jsonschema error should have all the info required, unless we want to wrap it in an error of our own?

Reply:
if it contains enough information for the user to fix the input, it's all good